cypress/integration/task-3.js

```diff
@@ -0,0 +1,46 @@
+import TimeLoggingPage from '../pom/timeLoggingPageObject'
+import NavBar from '../pom/navBarObject'
+
+const timeLoggingPage = new TimeLoggingPage()
+const navBar = new NavBar()
+
+describe('Task 2', function() {
```

Comment:
It is a bad practice to name something like that because after a few months or other people looking at your code might not be aware of what it mean

Reply:
Thanks for your feedback, 4 rows are highlighted which one are you referring to?